packages/diagrams/lib/src/align.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+import 'diagram_step.dart';
+
+class AlignDiagram extends StatelessWidget implements DiagramMetadata {
+  const AlignDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+//    List<Widget> diagramChildren;
+    const Widget logo = FlutterLogo(size: 60);
+    const Icon origin = Icon(
+      Icons.fiber_manual_record,
+      size: 20,
+    );
+    Widget heading;
+    Widget containerChild;
+    switch (name) {
+      case 'align_constant':
+        heading = const Text('Alignment.topRight');
+        containerChild = const Align(
+          alignment: Alignment.topRight,
+          child: logo,
+        );
+        break;
+      case 'align_alignment':
+        heading = const Text('Alignment Origin');
```

Comment:
Shouldn't this be "Alignment.center"? What are you trying to illustrate here, the center alignment constant, or the location of the origin?

Reply:
The location of the origin is the intention here. FractionlOffset and Alignemnt have different origins.